stable/redis/templates/deployment.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
```

Comment:
Given that this is in the deployment.yaml and not values.yaml I dont believe that is necessary. Namely the API between the chart dev and the user (the values.yaml) does not expose this.